ui/src/channels/HostConnection.tsx

```diff
@@ -85,7 +85,7 @@ export default function HostConnection({
             <Bullet16Icon
               className={cn(
                 'h-4 w-4 flex-none',
-                getHostConnectionColor(saga, status, matched)
+                `text-${getHostConnectionColor(saga, status, matched)}-400`
```

Comment:
@arthyn was this a concern for you? (concatenating Tailwind class names). 
This was already a value generated in runtime, the difference is it's being composed in this line now and between backticks, but still runtime, right?

Reply:
@arthyn Heard - take a look